homeassistant/components/zha/switch.py

```diff
@@ -138,3 +150,119 @@ async def async_update(self) -> None:
 
         self._state = len(on_states) > 0
         self._available = any(state.state != STATE_UNAVAILABLE for state in states)
+
+
+class ZHASwitchConfigurationEntity(ZhaEntity, SwitchEntity):
+    """Representation of a ZHA switch configuration entity."""
+
+    _attr_entity_category = EntityCategory.CONFIG
+    _zcl_attribute: str
+    _zcl_inverter_attribute: str = ""
+    _inverted = False
```

Comment:
Is this needed?

Reply:
forgot to remove `_inverted` and category should be defined in inherited class. one moment please